MultiLangProgGuide/MultiLangProgGuide.rst

```diff
@@ -1977,15 +1979,16 @@ Go BufferT Alloc()
 
 .. code-block:: go
 
-    Alloc(nBytes int)
+    Alloc(nBytes uint)
 
 Allocate:
 
-- a buffer in YottaDB heap space of size :code:`nBytes`;
-- a :code:`C.ydb_buffer_t` structure (also in C heap space)
-  referencing the buffer; and
-- a Go :code:`BufferT` structure whose :code:`cbuft` references the
-  :code:`C.ydb_buffer_t` structure.
+- a buffer in YottaDB heap space of size :code:`nBytes`; and
+- a :code:`C.ydb_buffer_t` structure, also in YottaDB heap space,
+  referencing the buffer.
+
+Set :code:`cbuft` in the :code:`BufferT`
+structure to reference the :code:`C.ydb_buffer_t` structure.
```

Comment:
Reference seems to be right, because cbuft is in Go memory but stores a pointer to a ydb_buffer_t structure in the YottaDB heap. So going from cbuft to the ydb_buffer_t would involve dereferencing a pointer.